web/client/epics/__tests__/featuregrid-test.js

```diff
@@ -1822,6 +1825,85 @@ describe('featuregrid Epics', () => {
         }));
     });
     describe('updateSelectedOnSaveOrCloseFeatureGrid', () => {
+        it("test savePendingFeatureGridChanges", (done) => {
```

Comment:
this test written this way is not testing really the changes you have done, it is only checking the initial saving action.

you need to extend the number of actions, by also testing the content of the payload, probably you have to mock some data